ws-nextjs-app/pages/[service]/new_live/[id]/Post/index.tsx

```diff
@@ -128,10 +162,11 @@ const Post = ({ post }: { post: PostType }) => {
   );
 
   const isBreakingNews = pathOr(false, ['options', 'isBreakingNews'], post);
+  const curated = pathOr('', ['dates', 'curated'], post);
```

Comment:
It might be worth renaming this to `timestamp` from `curated`, at first glance it doesn't give much context unless you look for dates in the path